src/hotspot/os/linux/os_linux.cpp

```diff
@@ -184,6 +184,10 @@ static bool suppress_primordial_thread_resolution = false;
 
 // utility functions
 
+julong os::Linux::oscontainer_cache_timeout() {
+  return NANOSECS_PER_SEC / OsCachedMetricsRefreshRate;
+}
+
```

Comment:
Shouldn't this be part of the `osContainer` API rather than `os::Linux`?

Reply:
Thanks! I moved it from os::Linux to OSContainer.